sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -163,6 +163,10 @@ abstract class Scrollable extends StatefulComponent {
     return scrollTo(newScrollOffset, duration: duration, curve: curve);
   }
 
+  void fling(Offset velocity) {
+    _handleDragEnd(velocity);
+  }
```

Comment:
So PageableList would override fling() instead of _handleDragEnd().  That seems OK to me.